compliance/tasks/main.yml

```diff
@@ -0,0 +1,232 @@
+---
+- name: ensure tmpfs mount options
+  mount: src=tmpfs name=/dev/shm fstype=tmpfs opts=rw,nodev,noexec,nosuid state=present
+  tags: [ compliance, tmpfs ]
+
+- name: Ensure additional security packages installed
+  yum:
+    name:
+      - aide
+      - psacct
+      - postfix
+    state: installed
+  tags: [ compliance, yum ]
+
+- name: Ensure additional security packages installed for Amazon Linux
```

Comment:
Based on the task name, should this have a `when: ansible_distribution == "Amazon"`?
It looks like these are selinux packages and not specific to Amazon Linux, so maybe just update the name.

Reply:
Verified with Tom that they should be valid for centos as well so I updated the name.